### DISCUSSION TITLE
Placeholder with full page component in V3.

### DISCUSSION THREAD

**Discussion by thatal** [livewire/livewire#6206] [1. Help] [answered]
Anyone please share how we can show placeholder for full page component during loading.

**Comment by ildyria** [comment]
Isn't this what you want: https://livewire.laravel.com/docs/lazy#rendering-placeholder-html

**Reply by thatal** [reply]
This is not for a full page component. PR already created https://github.com/livewire/livewire/pull/5780 but any alternate ?

**Comment by PhiloNL** [comment, marked as answer]
This is currently not supported, but as you mentioned, there is a PR for this.
 
You can achieve the same with something like this:

```php
class MyComponent extends Component {

    public $loadingDone = false;

    public function load()
   {
        // Perform heavy query etc.
        // ...

        $this->loadingDone = true;
   }
}
```

```blade
<div wire:init="load">
    @if($loadingDone)
        <div>Done loading</div>
    @else
        <div>Loading...</div>
    @endif
</div>
```

**Reply by thatal** [reply]
Thanks 🙏
